test/init_tracer.js

```diff
@@ -156,4 +155,23 @@ describe('initTracer', () => {
         assert.equal(tracer._metrics._factory, metrics);
         assert.equal(tracer._tags['x'], 'y');
     });
+
+    describe("upsampling", () => {
+        let config = {
+            serviceName: 'test-service'
+        };
+
+        it('should default to false', () => {
+            let tracer = initTracer(config);
+
+            assert.isNotOk(tracer._upsampling.enabled)
+        });
+
+        it('should pass true if set to true', () => {
+            config.upsampling = {"enabled" : true};
```

Comment:
Typically what I would recommend here is instead of mutating the shared config create a new config with the desired change:
```
const config = { ...config, upsampling: {"enabled" : true}}
```

(This is more relevant when the config is shared between more test cases.)